src/init.cpp

```diff
@@ -816,7 +816,7 @@ void InitParameterInteraction()
 
     // Warn if unrecognized section name are present in the config file.
     for (const auto& section : gArgs.GetUnrecognizedSections()) {
-        InitWarning(strprintf(_("Section [%s] is not recognized."), section));
+        InitWarning(strprintf(_("%s:%i:Section [%s] is not recognized."), section.file, section.line, section.name));
```

Comment:
Would it make sense to move the `%s:%i` out of the translation message to keep it the same? The translators aren't going to understand that part anyway.

Reply:
@laanwj Thank you  for your review. Addressed.